pkg/operator/ceph/nfs/ganesha.go

```diff
@@ -70,6 +75,19 @@ func (c *GaneshaController) createGanesha(n cephv1beta1.NFSGanesha) error {
 	return nil
 }
 
+// Create empty config file for new ganesha server
+func (c *GaneshaController) addRADOSConfigFile(n cephv1beta1.NFSGanesha, name string) error {
+	nodeID := getGaneshaNodeID(n, name)
+	config := getGaneshaConfigObject(nodeID)
+	err := c.context.Executor.ExecuteCommand(false, "", "rados", "--pool", n.Spec.RADOS.Pool, "--namespace", n.Spec.RADOS.Namespace, "stat", config)
+	if err == nil {
+		// If stat works then we assume it's present already
+		return nil
+	}
+	// try to create it
+	return c.context.Executor.ExecuteCommand(false, "", "rados", "--pool", n.Spec.RADOS.Pool, "--namespace", n.Spec.RADOS.Namespace, "create", config)
+}
```

Comment:
It would be nice if "rados create" returned a real error code when the object already exists , but it just returns 1. We could scrape the output, but blech. Another idea might be to roll some sort of special purpose program for this, but I'm not sure how we'd want to package and install it.